content/app/development/ux/fields/grouping/alternatives/_index.en.md

```diff
@@ -73,17 +83,111 @@ Eksempel:
 }
 ```
 
+## Vise gruppen som del av Panel
```

Comment:
Maybe this should be in english? Since we're working on translating everything anyway, we probably shouldn't continue adding to that workload.

Reply:
Oh yeah good point - I missed that in my review 😅